CHANGELOG.md

```diff
@@ -2,8 +2,10 @@
 
 ## [v5.0.0](https://github.com/ARMmbed/mbed-coap/releases/tag/v5.0.0)
 
-- Reduce heap footprint by storing only single block when receiving a blockwise message.
-    * User is now responsible of freeing the data by calling sn_coap_protocol_block_remove() and must not free the payload separately.
+- Reduce heap footprint by storing only single block when receiving a blockwise message. 
+    * This feature is behind a SN_COAP_REDUCE_BLOCKWISE_HEAP_FOOTPRINT flag which is disabled by default.
```

Comment:
This should be a new minor version. Call it 5.1.0

Reply:
Is it ok now?